nodenetworkconfig/manifests/acn.azure.com_nodenetworkconfigs.yaml

```diff
@@ -0,0 +1,104 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.3.0
+  creationTimestamp: null
+  name: nodenetworkconfigs.acn.azure.com
+spec:
+  group: acn.azure.com
+  names:
+    kind: NodeNetworkConfig
+    listKind: NodeNetworkConfigList
+    plural: nodenetworkconfigs
+    singular: nodenetworkconfig
```

Comment:
Could we add "nnc" as a short name like so?
```
shortNames:
      - nnc
```

Reply:
done